src/main/java/edu/gwu/csci6212/MinimumStepsChessKnight.java

```diff
@@ -0,0 +1,113 @@
+package edu.gwu.csci6212;
+
+
+import java.util.LinkedList;
+
+public class MinimumStepsChessKnight {
+
+    private int [][] board;
+    private int[] start;
+    private int[] goal;
+    private LinkedList<int[]> queue;
+    private boolean gameOver;
+    private int[][] knightMovements;
+    private int size;
+    private int totalSteps;
+
+    public MinimumStepsChessKnight(){
+        this.start = new int[3];
+        this.goal = new int[3];
+        this.queue = new LinkedList<>();
+        this.gameOver = false;
+
+        this.knightMovements = new int [][] {
```

Comment:
Can you explain what the step part of this triple is for? I would have expected a set of ordered pairs. Or perhaps an array of instances of a KnightMove class.